### ISSUE TITLE
Enhance animations

### ISSUE THREAD

**Issue by aakzsh** [aakzsh/zanie#8]
Right now, most of the animations of image widgets are happening through gifs, which doesnt look professional. We need to make them look more better quality and experience wise.
If you are willing to work on it, let us know in the comments and you'll be assigned the issue.

**Comment by agent515** [comment]
@aakzsh  I would like to take up this issue.

**Comment by aakzsh** [comment]
@agent515 that's cool! I'm assigning this issue to you, just a few pointers:-
all images are in assets folder
the demo video of how it was working before can be found here : https://youtu.be/_WjtL1Y-s7g

**Comment by agent515** [comment]
@aakzsh I have been very busy recently so could not spare time to work on it. But I have looked at the repo and have run the app.

I found GIF of genie coming out of the lamp.

So, is that what needs to be animated?

I'm not very familiar with animations, so it might be very tricky to pull off for me.

If you've some other animation ideas of your own to do in the app, it would be very helpful if you share that.

Thanks.
